lite/examples/sound_classification/android_compose/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/compose/ui/SoundClassifierScene.kt

```diff
@@ -0,0 +1,185 @@
+package org.tensorflow.lite.examples.soundclassifier.compose.ui
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material.*
```

Comment:
Use single imports

Reply:
Oh, I found that wildcard is not recommended by Kotlin style guide! Why Android Studio is doing this to me. XD (let's talk about this on chat)